db/migrate/20240722202054_create_oauth_token_exchange_grants.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+##
+# Creates the oauth_token_exchange_grants table.class CreateOAuthTokenExchangeGrants < ActiveRecord::Migration[7.0]
```

Comment:
Looks like the comment and class definition somehow got joined together. You'll need to fix like this:
```
##
#  Creates the oauth_token_exchange_grants table
class CreateOAuthTokenExchangeGrants < ActiveRecord::Migration[7.0]
  def change
    ...
  end
end
```